modules/core/src/main/java/org/apache/ignite/hlc/HybridClock.java

```diff
@@ -40,13 +47,25 @@ public class HybridClock {
     }
 
     /** Latest timestamp. */
-    private volatile HybridTimestamp latestTime;
+    protected volatile HybridTimestamp latestTime;
```

Comment:
I don't like an idea of adding test related code to HybridClock. Neither supplier nor protected or TestOnly. I'd rather extract HybridClock interface with several implementaions, e.g. common HybridClockImpl and TrackableHybridClock in test package. What do u think, will it work?